### ISSUE TITLE
Deletes all keys of maintained file

### ISSUE THREAD

**Issue by emkayy** [GertjanReynaert/react-intl-translations-manager#171]
Hi there,

this is my setup:

```
manageTranslations({
  messagesDirectory: 'i18n/lang/',
  translationsDirectory: 'i18n/compiled-lang/',
  whitelistsDirectory: 'i18n/whitelist/',
  languages: ['en']
});
```
If I run it, it will just delete all the key in the maintained en.json file.
Even if all the translations are the same as in the default language, it won't give me an untranslated warning.

Can anyone help?

**Comment by tgamauf** [comment]
Same here, always creates an empty json file for the translated messages and says everything is alright.

**Comment by AndrzejSuwalaKeyloop** [comment]
Yes, I have the same issue - always empty file as a result. Deletes everything.
